src/main/java/dev/goral/rpgmanager/rpgSystems/cpRed/characters/CpRedCharactersDTO.java

```diff
@@ -0,0 +1,51 @@
+package dev.goral.rpgmanager.rpgSystems.cpRed.characters;
+
+import lombok.Getter;
+
+@Getter
+public class CpRedCharactersDTO {
+    private final Long id;
+    private final Long gameId;
+    private final Long userId;
+    private final String name;
+    private final String nickname;
+    private final String type;
+    private final Integer expAll;
+    private final Integer expAvailable;
+    private final Integer cash;
+
+    public CpRedCharactersDTO(Long id,
+                              Long gameId,
+                              Long userId,
+                              String name,
+                              String nickname,
+                              String type,
+                              Integer expAll,
+                              Integer expAvailable,
+                              Integer cash) {
+        this.id = id;
+        this.gameId = gameId;
+        this.userId = userId;
+        this.name = name;
+        this.nickname = nickname;
+        this.type = type;
+        this.expAll = expAll;
+        this.expAvailable = expAvailable;
+        this.cash = cash;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
wgl z tego co wiem to też jest w lomboku `@ToString`, więc możesz zobaczyć jak to wygląda i ewentualnie podmienić